eventlet/support/greendns.py

```diff
@@ -398,10 +399,12 @@ def getaliases(self, hostname):
 resolver = ResolverProxy(hosts_resolver=HostsResolver())
 
 
-def resolve(name, family=socket.AF_INET, raises=True, _proxy=None):
+def resolve(name, family=socket.AF_INET, raises=True, _proxy=None,
+            use_dns=True):
```

Comment:
Unless this name is commonly known in other APIs, could you please rename it to `use_network`?

Reply:
ack